src/private/MDMPresentationTransitionController.m

```diff
@@ -93,10 +93,14 @@ - (UIPresentationController *)presentationControllerForPresentedViewController:(
     return nil;
   }
   id<MDMTransitionWithPresentation> withPresentation = (id<MDMTransitionWithPresentation>)_transition;
-  _presentationController = [withPresentation presentationControllerForPresentedViewController:presented
-                                                                      presentingViewController:presenting
-                                                                          sourceViewController:source];
-  return _presentationController;
+  UIPresentationController *presentationController =
+      [withPresentation presentationControllerForPresentedViewController:presented
+                                                presentingViewController:presenting
+                                                    sourceViewController:source];
+  // _presentationController is weakly-held, so we have to do this local var dance to keep it
+  // from being nil'd on assignment.
```

Comment:
My understanding is that weak pointers won't be auto-nil'd in the middle of a method... is that what you're saying here?